velox/exec/RowContainer.cpp

```diff
@@ -364,6 +366,18 @@ int32_t RowContainer::findRows(folly::Range<char**> rows, char** result) {
   return numRows;
 }
 
+void RowContainer::appendNextRow(char* current, char* nextRow) {
+  NextRowVector*& nextRowArrayPtr = getNextRowVector(current);
+  if (!nextRowArrayPtr) {
+    nextRowArrayPtr =
+        new NextRowVector(StlAllocator<char*>(stringAllocator_.get()));
```

Comment:
There is a caveat that `NextRowVector` itself is allocated directly from malloc and not counted in pool.  Can we fix this?

Reply:
Do you have any suggestion for this issue? For example we implment the vector self as you mentioned in before comments, then we allocate vector self and data array both from `stringAllocator_`.
Thanks.

Reply:
You can allocate the memory using `HashStringAllocator::allocate`, then call the constructor on it using `new` with placement; when release it you need to call `HashStringAllocator::free` instead of `delete`.